tests/snappi_tests/pfc/files/helper.py

```diff
@@ -129,6 +130,9 @@ def run_pfc_test(api,
         # PFC pause frame capture is not requested
         valid_pfc_frame_test = False
 
+    pause_flow_dur_sec = DATA_FLOW_DURATION_SEC + data_flow_delay_sec + SNAPPI_POLL_DELAY_SEC + \
+        PAUSE_FLOW_DUR_BASE_SEC if valid_pfc_frame_test else 'continuous'
+
```

Comment:
If `valid_pfc_frame_test` is `False`, then you are adding a string to some integers.

Reply:
no this is just setting the mode to be fixed when this duration is passed in

Reply:
changed